plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.test.tsx

```diff
@@ -94,15 +94,15 @@ describe('<RecentWorkflowRunsCard />', () => {
     );
 
   it('renders a table with a row for each workflow', async () => {
-    const subject = await renderSubject();
```

Comment:
As a bonus, I've also removed these unnecessary errant `await`s, as `renderSubject` is not async :)